draft-demarco-oauth-status-assertions.md

```diff
@@ -596,7 +596,11 @@ the endpoint where the Wallet Instances can request Status Assertions.
 the Wallet Instance to hash the Digital Credential for which the
 Status Assertion is requested,  using one of the hash algorithms listed
 in the [IANA-HASH-REG].
+- `credential_status_detail_supported`. OPTIONAL. JSON array that outlines the details of each Credential's validity status supported by the Credential Issuer. This parameter MAY be used to extend the values defined in Section [Status Assertion](#status-assertion). Each entry MUST contain the following parameters:
 
+    - `credential_status_validity`. Numerical value indicating the validity of the Credential.  
+    - `state`. String value of a Credential status supported.
+    - `description`. String containing the human-readable description of the status related to this object.
```

Comment:
@marinaado , I added this additional optional parameter in the metadata of a CI. The idea is that a CI MAY use this parameter to extend the values defined within this specification (0, 1, 2) and the Wallet can obtain from the CI's Metadata the detailed info to prompt the User.  
WDYT about this? 

@OR13 , @peppelinux  ^^

Reply:
Agreed that the IANA registry reserved range for private use should be used to communicate values outside of the ones defined in the IANA registry.